packages/babel-helper-member-expression-to-functions/src/index.js

```diff
@@ -100,6 +100,26 @@ const handle = {
       return;
     }
 
+    // { KEY: MEMBER } = OBJ -> { KEY: _ref } = OBJ; _set(MEMBER, _ref);
+    // [MEMBER] = ARR -> [_ref] = ARR; _set(MEMBER, _ref);
+    if (
```

Comment:
If I remember correct, this will affect `super.foo` expressions, too. Can you add a test for assigning `[super.foo] = array`, too?

Reply:
where do you think should i add the test for `[super.foo] = array` ?

in `babel-plugin-transform-classes/test/get-set` ?